src/exercise/03-extra-1.html

```diff
@@ -0,0 +1,17 @@
+<body>
+  <div id="root"><div className="container">Hello World!</div></div>
+  <script src="https://unpkg.com/react@18.1.0/umd/react.development.js"></script>
+  <script src="https://unpkg.com/react-dom@18.1.0/umd/react-dom.development.js"></script>
+
+  <script src="https://unpkg.com/@babel/standalone@7.12.4/babel.js"></script>
+
+  <script type="text/babel">
+    const children = 'Hello World!!'
+    const className = 'container'
+    const props = {children, className}
+
+    const element = <div {...props} />
```

Comment:
I usually don't recommend using spread operator with props as you might not now what extra props you are sending to the element.

In this case is good thou

Reply:
Yeah I agree its better to be explicit 👍